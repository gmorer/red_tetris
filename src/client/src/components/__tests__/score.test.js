import Score from './../score';
import React from 'react';
import renderer from 'react-test-renderer';

test('Some test', () => {
	const component = renderer.create(<Score score="6" />)
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
})