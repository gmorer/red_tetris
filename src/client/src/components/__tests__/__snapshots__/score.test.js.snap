// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some test 1`] = `
<div
  style={
    Object {
      "backgroundColor": "rgba(159, 35, 58, 0.5)",
      "height": "5em",
      "width": "10em",
    }
  }
>
  Score : 
  6
</div>
`;
